test/integration/targets/route53/aliases

```diff
@@ -1,2 +1,2 @@
 cloud/aws
-unsupported
+shippable/aws/group3
```

Comment:
I thought there is no `group3` (yet)?

Reply:
Indeed there is not. I was thinking about the *nix groups.